ppa/archive/admin.py

```diff
@@ -10,10 +10,20 @@ class DigitizedWorkAdmin(admin.ModelAdmin):
         'added', 'updated')
     search_fields = ('source_id', 'title', 'author', 'enumcron', 'pub_date',
         'publisher')
+    filter_horizontal = ('collections',)
     # date_hierarchy = 'added'  # is this useful?
-    # currently nothing useful to filter on; eventually will have collectio
-    # list_filter = []
+    list_filter = ['collections']
 
+    def save_related(self, request, form, formsets, change):
+        '''Ensure reindex is called when admin form is saved'''
+        # m2m relations are handled separately by the form so the standard
+        # save override will not help
```

Comment:
On this one, perhaps only the source code for `save_related` itself. The behavior of the admin save functionality isn't entirely obvious until you hit the problem I did (i.e. save is called with m2m links). I will include a link to it, because its doc string comes as close to a single explanation of the issue as I have seen.

Reply:
Documentation added.